content/en/blog/2022/k8sattributes.md

```diff
@@ -0,0 +1,252 @@
+---
+title:
+  Improved application & infrastructure troubleshooting attaching k8s attributes
+linkTitle: Attaching Kubernetes attributes to traces
```

Comment:
Ok, we're making progress on the title & short title. I'd suggest the following adjustments:

- title: Improved troubleshooting using k8s attributes
- linkTitle: Kubernetes attributes

Thoughts @svrnm @rubenvp8510?

Reply:
I like, make sense to me

Reply:
sounds good to me!